code/modules/mob/living/carbon/xenomorph/castes/puppeteer/abilities_puppeteer.dm

```diff
@@ -0,0 +1,449 @@
+// ***************************************
+// *********** Flay
+// ***************************************
+/datum/action/xeno_action/activable/flay
+	name = "Flay"
+	action_icon_state = "flay"
+	desc = "Takes a chunk of flesh from the victim marine through a quick swiping motion, adding 100 biomass to your biomass collection."
+	ability_name = "flay"
+	plasma_cost = 0
+	cooldown_timer = 20 SECONDS
+	target_flags = XABB_MOB_TARGET
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_FLAY,
+	)
+
+/datum/action/xeno_action/activable/flay/can_use_ability(atom/target, silent = FALSE, override_flags)
+	. = ..()
+	if(!ishuman(target))
+		if(!silent)
+			to_chat(owner, span_xenowarning("We cant find any suitable flesh on this thing!"))
```

Comment:
balloon alerts?

Reply:
too long for a balloon alert

Reply:
Shorten the text
eg here
"not human"
or
"no flesh"
etc